### ISSUE TITLE
Is something updated?

### ISSUE THREAD

**Issue by yangming915** [carefree0910/carefree-creator#23]
Last day I launch my local server. when it load an init the model, it failed! It looks like it need to use the new file about prompt enhance API, the console show that it is a 'pytorch_model.bin' file with 334M and need to download it. But as you know, it very slow to do that even connectError. So I try to download it on the 'huggingface', there is some updated file with 334Mb seem right. 
But when I download it from the page, I got a 318M one like the local old one , I certainly think it is the old one after i use it.
I want to use the new one, how can I get it?

**Comment by carefree0910** [comment]
Hi!

The `prompt enhance API` is indeed a new feature, which directly borrows from [here](https://huggingface.co/FredZhang7/distilgpt2-stable-diffusion-v2), so technically speaking you can directly download it yourself!

And since I'm just using that API, I don't know whether it has old/new versions. 😣

**Comment by yangming915** [comment]
yes I know this page, it is so troubling I want to download the new one but it always give me the old one. The new one with 334Mb size seem to be updated last day, but it seems anything wrong.

**Comment by yangming915** [comment]
> Hi!
> 
> The `prompt enhance API` is indeed a new feature, which directly borrows from [here](https://huggingface.co/FredZhang7/distilgpt2-stable-diffusion-v2), so technically speaking you can directly download it yourself!
> 
> And since I'm just using that API, I don't know whether it has old/new versions. 😣

And can I know the audit_algorithm is in my local sever or your cloud?

**Comment by yangming915** [comment]
> > Hi!
> > The `prompt enhance API` is indeed a new feature, which directly borrows from [here](https://huggingface.co/FredZhang7/distilgpt2-stable-diffusion-v2), so technically speaking you can directly download it yourself!
> > And since I'm just using that API, I don't know whether it has old/new versions. 😣
> 
> And can I know the audit_algorithm is in my local sever or your cloud?

thanks for your reply

**Comment by carefree0910** [comment]
Hi!

The audit algorithms are on my cloud, after all if it wasn't for the policy restrictions from the cloud I will not bother using audit algorithms anyway! 🤣

So if you deployed locally and use it with API calls, no audit will be performed; but if you are using the creator product, I'll have to apply the audit algorithms (because either you are using my cloud to calculate, or you are storing the generated images on my cloud.)

**Comment by yangming915** [comment]
> Hi!
> 
> The audit algorithms are on my cloud, after all if it wasn't for the policy restrictions from the cloud I will not bother using audit algorithms anyway! 🤣
> 
> So if you deployed locally and use it with API calls, no audit will be performed; but if you are using the creator product, I'll have to apply the audit algorithms (because either you are using my cloud to calculate, or you are storing the generated images on my cloud.)

haha, I really want to use your webUI, it is convenient. Thanks again.

**Comment by carefree0910** [comment]
In case you are interested: [this issue](https://github.com/carefree0910/carefree-creator/issues/22#) also discussed about the audit stuffs!
